daemon/api_apps.go

```diff
@@ -214,8 +214,13 @@ func getLogs(c *Command, r *http.Request, user *auth.UserState) Response {
 		serviceNames[i] = appInfo.ServiceName()
 	}
 
+	// Include journal namespaces if supported. The --namespace option was
+	// introduced in systemd version 245. If systemd is older than that then
+	// we cannot use journal quotas in any case and don't include them.
+	version, err := systemd.Version()
+	includeNamespaces := err == nil && version >= 245
```

Comment:
Is it OK to not act on the error in retrieving the systemd version and just get logs without the namespaces? Should I as a minimum log it?

Reply:
IMHO in this case it's fine as it is. If you want to check the error, then maybe it's better to use something like
```go
includeNamespaces := false
if err := systemd.EnsureAtLeast(245); err == nil {
    includeNamespaces = true
} else if !systemd.IsSystemdTooOld(err) {
    return fmt.Errorf("cannot get systemd version: %v", err)
}```
But it might be overkill. If, as you wrote, journal quotas don't work in versions older than 245, then I see little point in doing this check.

Reply:
I like your suggestion a lot, I'll update the code to be like that.